daemon/api_quotas.go

```diff
@@ -68,6 +70,10 @@ var (
 	servicestateRemoveQuota = servicestate.RemoveQuota
 )
 
+var getQuotaMemUsage = func(grp *quota.Group) (quantity.Size, error) {
```

Comment:
(nitpick) most of the other code that uses a mocking style like this is written:
```
// grpCurrentMemoryUsage is a var so that it can be mocked in tests.
var grpCurrentMemoryUsage = grp.CurrentMemoryUsage
```
(maybe no comment, we rarely have them when I look at e.g. boot/seal.go where we do this quite a bit for the secboot stuff).

Reply:
hmm how does that snippet you provided work though? grp is an instance of an object? 

perhaps I have misunderstood your suggestion ? https://play.golang.org/p/NMRVN1ZGKuO

```
invalid method expression grp.CurrentMemoryUsage (needs pointer receiver: (*quota.Group).CurrentMemoryUsage)
```

Reply:
Oh, silly me, please ignore me.